### PR TITLE
Team stats tests

### PR DESCRIPTION
**Pull request by hsmitha26**
Tests created for Iteration 4.  Just need data for the last test.

### CODE REVIEW

File: test/team_stats_test.rb [hunk 1]
```diff
@@ -0,0 +1,178 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+require './lib/stat_parser'
```
Comment (wipegup) [inline test/team_stats_test.rb]: One thing I did was to require 'stat_parser' in the 'stat_tracker' class -- so it should not be necessary to require it here

File: test/team_stats_test.rb [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    end
+
+  def test_team_info_with_respective_team_attributes
+    expected = {
+                team_id: 3,
+                franchise_id: 10,
+                short_name: "NY Rangers",
+                team_name: "Rangers",
+                abbreviation: "NYR",
+                link: "/api/v1/teams/3"
```
Comment (wipegup) [inline test/team_stats_test.rb:32]: At the moment, I believe that the ONLY info we have saved for teams are: `team_id` and `team_name` -- It looks like we should probably save some more.

File: test/team_stats_test.rb [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                }
+    assert_equal expected, @v_small_data.team_info
```
Comment (wipegup) [inline test/team_stats_test.rb]: Should team_info take an argument? Like a `team_id` or `team_name` ?

File: test/team_stats_test.rb [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+  end
+
+  def test_worst_season
+    assert_equal 20122013, @v_small_data.worst_season(3)
+    assert_equal 20122013, @small_data.worst_season(16)
+    assert_equal 20142015, @data.worst_season(5)
+  end
+
+  def test_average_win_percentage
+    assert_equal 1.0, @v_small_data.average_win_percentage(6)
```
Comment (wipegup) [inline test/team_stats_test.rb:70]: I know that I organized the data- aggregations by team_id -- However, I wonder if organizing them by teamName might be more readable

File: test/team_stats_test.rb (hunk 2 above)
Comment (jalena-penaligon) [reply test/team_stats_test.rb:32]: Yes, we realized when Brian posted the Spec Harness that they did want to see the full team info for this one. I took a stab at updating the conversions module (and StatParser/conversions_test) to no longer delete these keys and sent you a Pull Request!

**Comment by hsmitha26** [comment]
Updates are ready based on feedback. Thanks, Will!

**Pull request merged**